build/azDevOps/packages-amido-stacks-configuration.yml

```diff
@@ -34,14 +41,20 @@ variables:
     value: "$(Package.Path).Tests"
   - name: BuildPipeline.Path
     value: "build/azDevOps/packages-amido-stacks-configuration.yml"
+  # Yamllint
+  - name: yamllint_config_file
+    value: "${{ variables.self_repo_dir }}/yamllint.conf"
+  - name: yamllint_scan_directory
+    value: "."
 
-#Trigger paths specific for this package, change accordingly
+# Trigger paths specific for this package, change accordingly
 trigger:
   branches:
     include:
-      - main
-      - feature/*
-      - bug*
+    - 'main'
+  paths:
+    include:
+    - '*'
```

Comment:
Any reason you've added this back in?  I believe this is the default settings.

Reply:
There were some odd branches specified before, I've set it back to defaults here.

Reply:
the branches part I get, but this is about the paths. 

> If you don't set path filters, then the root folder of the repo is implicitly included by default. 

https://docs.microsoft.com/en-us/azure/devops/pipelines/repos/github?view=azure-devops&tabs=yaml#ci-triggers

Reply:
Yeah, I'm just setting the default, being explicit, it's how we've gone it everywhere else.